socketshark/receiver.py

```diff
@@ -66,6 +69,7 @@ async def ping_handler(self):
                     # Ping timeout
                     ping.cancel()
                     self.shark.log.warn('redis ping timeout')
+                    self._stop = True
```

Comment:
So now, if the ping to ANY of the N Redises times out, we stop the whole receiver? Does that mean that, if we were to listen to N Redises and we shut one of them down, the whole SocketShark stops receiving messages published on all the other running Redises?

Reply:
That's correct. As I said in `_redis_connection_handler`, the goal is to abort on a Redis connection error so clients quickly reconnect to another host. You make a good point that losing one Redis server can take down the whole app. This currently makes it less valuable for permanent multi-redis configurations.

Reply:
Gotcha. In such case we need to remember (and document somewhere) that, during migrations from one Redis to another, you:
* CANNOT do this:
    1. Have a SocketShark service that's connected to Redis A.
    2. Set up a Redis B.
    3. Make the SocketShark service connect to Redis A & Redis B.
    4. Change other services to start publishing to Redis B.
    5. Shut down Redis A.
        * This would make the whole SocketShark service stop.
* CAN do this:
    1. Same steps 1-4 as above.
    2. Update the SocketShark service to only connect to Redis B.
    3. Shut down Redis A.

Right?

Reply:
Yes, exactly. I extended the comment on the `REDIS_ALT` configuration to make this clear.

a415332